doc/classes/ResourceImporterScene.xml

```diff
@@ -65,6 +65,9 @@
 		<member name="nodes/root_scale" type="float" setter="" getter="" default="1.0">
 			The uniform scale to use for the scene root. The default value of [code]1.0[/code] will not perform any rescaling. See [member nodes/apply_root_scale] for details of how this scale is applied.
 		</member>
+		<member name="nodes/root_script" type="Script" setter="" getter="" default="null">
+			If set to a valid script, attaches the script to the root node of the imported scene. If the type of the root node is not compatible with the script, the root node will be replaced with a type that is compatible with the script. This setting can also be used on other non-mesh nodes in the scene to attach scripts to them.
```

Comment:
Needs a rebase.

Also:

> If the type of the root node is not compatible with the script, the root node will be replaced with a type that is compatible with the script.

What does this mean in practice? Will I lose my gdscript?